app/views/content/github/_review_checklist.erb

```diff
@@ -25,11 +24,13 @@
 - [ ] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
 - [ ] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
 - [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
-- [ ] **Automated tests:** Are there automated tests or manual steps described so that the function of the software can be verified?
+- [ ] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
 - [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support
 
 ### Software paper
 
-- [ ] **Authors:** Does the `paper.md` file include a list of authors with their affiliations?
+- [ ] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
 - [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
-- [ ] **References:** Do all archival references that should have a DOI list one (e.g., papers, datasets, software)?
+- [ ] **State of the field:** Do the authors describe how this software compares to other available packages? Are key relevant or alternative approaches cited and briefly discussed?
```

Comment:
I'm a bit dissatisfied with this.

For software that was new, this might be a useful point in understanding if people would cite it, but for software that is old and well-used but is not really distinct from other software, what would the authors say?

And I fear this could lead to the author saying something, the reviewer saying "you also should mention X", but still not being complete.

I wonder if we could change it to more simply:

**State of the field:** Do the authors describe how this software compares to other commonly-used packages?

Reply:
What do you think @Kevin-Mattheus-Moerman?

Reply:
I see what @danielskatz is saying. His proposed shorter/simpler form sounds fine too.